src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.scss

```diff
@@ -0,0 +1,43 @@
+@import "scss/constants.scss";
+
+.ScheduleVenueDescription {
+  flex: 1;
+  padding-left: 20px;
+
+  &__main {
+    display: flex;
+    align-items: center;
+    margin-top: 2rem;
+    margin-bottom: 1rem;
+  }
+
+  &__pic {
+    flex-shrink: 0;
+    width: 92px;
+    height: 92px;
+    background-size: cover;
+    border-radius: 46px;
+    margin-right: 20px;
+    cursor: pointer;
+    transition: all 400ms $transition-function;
+    box-shadow: 0 8px 16px rgba($black, 0.3);
```

Comment:
same as in the styles above. Please, check Posterhall, PosterPage venue templates for a reference

Reply:
applied variables: https://github.com/sparkletown/sparkle/pull/1302/commits/06620c6fb5ce10998508b5403d8b8cf907ed32d6